wasm.md

```diff
@@ -920,9 +922,9 @@ Unlike labels, only one frame can be "broken" through at a time.
     syntax Instr ::= "(" "invoke" Int ")"
```

Comment:
Put some documentation for `invoke` in the markdown above this. Explain why the double reverse is here.